otterdog/eclipse-score.jsonnet

```diff
@@ -4,6 +4,7 @@ orgs.newOrg('eclipse-score') {
   settings+: {
     has_discussions: true,
     discussion_source_repository: "eclipse-score/score",
+    web_commit_signoff_required: true,
```

Comment:
setting explicit value here is not needed, as this is already the default, you can also check that here: https://otterdog.eclipse.org/projects/automotive.score#settings

Reply:
you can see all defaults here: https://otterdog.eclipse.org/projects/automotive.score/defaults

Reply:
What is the policy for changing the defaults in otterdog? Will all repos using otterdog be updated with the "old" default or silently migrated?

Reply:
ofc, if we would ever change a default setting we would make sure that all configurations are updated accordingly.

Reply:
take the defaults as a reasonable value to start with. The less customizations you have the better. Some values are controversial, e.g. merge commits vs squash merges, so chosing a default there is hard.

Also there is the option to define your own custom templates that you use, take a look at https://github.com/eclipse-daanse/.eclipsefdn/blob/main/otterdog/eclipse-daanse.jsonnet

Reply:
OK thanks, I think it's fine for now to keep the otterdog's defaults.